control-plane/config/crd/bases/mesh.consul.hashicorp.com_gatewayclassconfigs.yaml

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.12.1
+  name: gatewayclassconfigs.mesh.consul.hashicorp.com
+spec:
+  group: mesh.consul.hashicorp.com
+  names:
+    kind: GatewayClassConfig
+    listKind: GatewayClassConfigList
+    plural: gatewayclassconfigs
+    shortNames:
+    - gateway-class-config
+    singular: gatewayclassconfig
+  scope: Namespaced
```

Comment:
I think we wanted this to be `Cluster`-scoped. We just now realized this. Will you add the following annotation to the `GatewayClassConfig`? `// +kubebuilder:resource:scope=Cluster`

Reply:
You got it